clippy_lints/src/utils/conf.rs

```diff
@@ -106,6 +106,9 @@ macro_rules! define_Conf {
 
 pub use self::helpers::Conf;
 define_Conf! {
+    /// Lints: MANUAL_NON_EXHAUSTIVE, MANUAL_STRIP, OPTION_AS_REF_DEREF, MATCH_LIKE_MATCHES_MACRO. The minimum rust version 
+    /// that the project supports
+    (msrv, "msrv": Option<String>, None),
```

Comment:
I think we can just use `String` here. A good default then is `1.0.0`.

Reply:
Sounds good!